src/m365/spo/commands/group/group-member-remove.ts

```diff
@@ -125,10 +118,78 @@ class SpoGroupMemberRemoveCommand extends SpoCommand {
       });
 
       if (result.continue) {
-        await removeUserfromSPGroup();
+        await this.removeUserfromSPGroup(logger, args);
       }
     }
   }
+
+  private async removeUserfromSPGroup(logger: Logger, args: CommandArgs): Promise<void> {
+    if (this.verbose) {
+      logger.logToStderr(`Removing User with Username ${args.options.userName} from Group: ${args.options.groupId ? args.options.groupId : args.options.groupName}`);
+    }
+
+    let requestUrl: string = `${args.options.webUrl}/_api/web/sitegroups/${args.options.groupId
+      ? `GetById('${args.options.groupId}')`
+      : `GetByName('${formatting.encodeQueryParameter(args.options.groupName as string)}')`}`;
+
+    if (args.options.userName) {
+      const loginName: string = `i:0#.f|membership|${args.options.userName}`;
+      requestUrl += `/users/removeByLoginName(@LoginName)?@LoginName='${formatting.encodeQueryParameter(loginName)}'`;
+    }
+    else {
+      const aadGroupId = await this.getGroupId(args);
+      requestUrl += `/users/RemoveById(${aadGroupId})`;
+      logger.log(aadGroupId);
```

Comment:
I think we should remove this one. Otherwise, the command will output the aad group id when we remove by name or id which I feel is rather unexpected 😉